parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -73,6 +74,8 @@ ERR_AZURE_STACK_GET_SUBNET_PREFIX=122 {{/* Error fetching the subnet address pre
 ERR_SWAP_CREAT_FAIL=130 {{/* Error allocating swap file */}}
 ERR_SWAP_CREAT_INSUFFICIENT_DISK_SPACE=131 {{/* Error insufficient disk space for swap file creation */}}
```

Comment:
Oh yea, good point. My only concern with changing already-set CSE exit codes is that it might throw off debugging efforts for our on-calls if the DRI isn't aware of the change.